refactor: Update OAuth2 token URL in UTMSP product JSON

govindsi opened the pull request:
The OAuth2 token URL in the UTMSP product JSON has been updated with dummy sample data.

govindsi committed:
refactor: Update OAuth2 token URL in UTMSP product JSON

The OAuth2 token URL in the UTMSP product JSON has been updated
with dummy sample data.

Signed-off-by: Govind Singh <govind.sk85@gmail.com>

hrishiballal commented on qgcs-config/product.json:
```diff
@@ -8,6 +8,11 @@
     },
+    "utmsp_oauth2_data": {
+        "token_url": "https://utmsp.com/oauth/token",
+        "client_id": "client_id",
```
The Client ID and especially the Client Secret cannot be in the `product.json` file, we can discuss this in this week's call and finalize

govindsi committed:
Remove client_id and client_secret from product.json

client_id and client_secret will be supplied from runtime
environment configuration.

Signed-off-by: Govind Singh <govind.sk85@gmail.com>
